crates/services/consensus_module/poa/src/pre_confirmation_signature_service.rs

```diff
@@ -173,6 +180,7 @@ where
                 TaskNextAction::Stop
             }
             res = self.tx_receiver.receive() => {
+                tracing::debug!("AURELIEN: Res is {:?}", &res);
```

Comment:
stray trace?

Reply:
Yeah  when all tests pass I remove it. I should have added a comment, but all tests pass so removing it.